Makefile

```diff
@@ -18,20 +18,22 @@ install:
 dist:
 	@mkdir dist &&\
 		GOOS=linux GOARCH=amd64 go build -o "dist/testtrack.linux" ${LDFLAGS} github.com/Betterment/testtrack-cli/testtrack &&\
-		GOOS=darwin GOARCH=amd64 go build -o "dist/testtrack.darwin" ${LDFLAGS} github.com/Betterment/testtrack-cli/testtrack
+		GOOS=darwin GOARCH=amd64 go build -o "dist/testtrack.darwin-amd64" ${LDFLAGS} github.com/Betterment/testtrack-cli/testtrack &&\
+		GOOS=darwin GOARCH=arm64 go build -o "dist/testtrack.darwin-arm64" ${LDFLAGS} github.com/Betterment/testtrack-cli/testtrack
 
 release: distclean dist
 	@hub release create\
 		-a dist/testtrack.linux\
-		-a dist/testtrack.darwin\
+		-a dist/testtrack.darwin-amd64\
+		-a dist/testtrack.darwin-arm64\
```

Comment:
we will have to upgrade our home-brew formula to pull the right one for the current architecture